app/tasks/maintenance/policy_announcement_email_task.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Maintenance::PolicyAnnouncementEmailTask < MaintenanceTasks::Task
+  def collection
+    User.all
```

Comment:
Limit to users with confirmed emails?

Reply:
I thought so too, but will need to ask this question up the chain.

Reply:
Legal has come with the recommendation that we email everyone. Will need to update the task to handle locked users as well